Reflection/Timing.simba

```diff
@@ -1,34 +1,76 @@
-type
-  Timer = record
-    Tick: Int64;
-  end;
+{$include_once Internal/Reflection.simba}
+{$include_once Constants.simba}
+{$include_once srl/shared.simba}
 
-Procedure Timer.Start;
+{*
+R_TileToMM
+~~~~~~~~~~
+Converts the input tile to a point on the minimap.
+
+Example:
+  Mouse.Move(R_TileToMM(R_GetMe.Tile));
+*}
+Function R_TileToMM(Tile: TPoint): TPoint;
 begin
-  tick := GetTickCount();
+  Result := RSTile(Tile).ToLocal.TileToMM;
 end;
 
-Function Timer.ElapsedTime: Int64;
+{*
+R_TileToMM
+~~~~~~~~~~
+Converts the input point to a mainscreen point.
+
+Example:
+  Mouse.Move(R_MMToMS(R_TileToMM(R_GetMe.Tile)));
+*}
+Function R_MMToMS(P: TPoint): TPoint;
 begin
-  Result := GetTickCount() - self.Tick;
+  Result := TPoint(RSTile(P).MMToTile.TileToMS);
 end;
 
-Procedure Timer.Reset();
+{*
+R_TileHeight
+~~~~~~~~~~~~
+Returns the height of a given tile.
+
+Example:
+  if (R_TileHeight > 13) then
+    ...
+*}
+Function R_TileHeight(Tile: TPoint): Int32;
 begin
-  self.Tick := 0;
+  Result := RSTile.GetHeight;
```

Comment:
This is a bug. `RSTile` is not defined here. The parameter is `Tile`. It should be an `RSTile` as the parameter?

Reply:
Updated and fixed.